### ISSUE TITLE
[BITV] 9.3.3.2/2.2 - Set location for weather - The input field "Set custom address" does not have a visible label, and adjacent elements do not explain the function. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#37084]
see also https://github.com/nextcloud/server/issues/37091

- [x] https://github.com/nextcloud/server/pull/39971

![51e36c48b52ba8aa27249843b98fe342](https://user-images.githubusercontent.com/1315170/223561229-81e951f4-addc-4d27-bce1-69076855b574.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-9f92015c98-v2-n2

</details>
